src/axom/slam/BivariateMap.hpp

```diff
@@ -141,6 +188,76 @@ class BivariateMap
     , m_map(SetType(bSet->size()), defaultValue, stride, allocatorID)
   { }
 
+  /// \overload
+  template <typename UBSet,
+            typename TBSet = BivariateSetType,
```

Comment:
Just "type". We might be able to get rid of that parameter and use `BivariateSetType` directly since we're relying on `UBSet` as the SFINAE-able template argument.